python/src/lib.rs

```diff
@@ -45,6 +46,13 @@ use fragment::{DataFile, FileFragment};
 pub use reader::LanceReader;
 pub use scanner::Scanner;
 
+use crate::executor::BackgroundExecutor;
+
+// TODO: make this runtime configurable (e.g. num threads)
+lazy_static! {
```

Comment:
No need to change this now but `once_cell` seems to be preferred over `lazy_static` (and `once_cell` is on its way into the standard):

https://github.com/bluejekyll/trust-dns/pull/1944

Reply:
Good to know!